convert_csv_to_mt.py

```diff
@@ -123,75 +129,78 @@ def __init__(self, uniBars, outputPath):
             bars += pack('<d', uniBar[3])                               # High
             bars += pack('<d', uniBar[2])                               # Low
             bars += pack('<d', uniBar[4])                               # Close
-            bars += pack('<Q', int(uniBar[5]*1e6))                      # Volume TODO
-            bars += pack('<i', round((uniBar[3]) - uniBar[2])*10000)    # Spread TODO
-            bars += pack('<Q', int(uniBar[5]*1e6))                      # Real volume TODO
+            bars += pack('<Q', int(uniBar[5]*1e6))                      # Volume TODO ?
+            bars += pack('<i', round((uniBar[3]) - uniBar[2])*10000)    # Spread TODO ?
+            bars += pack('<Q', int(uniBar[5]*1e6))                      # Real volume TODO ?
 
         self._write(header + bars, outputPath)
 
 
 class FXT(Output):
     def __init__(self, uniBars, spread, outputPath):
-        # Build header (148 Bytes in total)
-        binVersion      = pack('<i', 403)
-        binCopyright    = bytearray('Copyright 2001-2015, MetaQuotes Software Corp.'.ljust(64, '\x00'), 'latin1', 'ignore')
-        binSymbol       = bytearray(symbol.ljust(12, '\x00'), 'latin1', 'ignore')
-        binPeriod       = pack('<i', timeframe)
-        binModel        = pack('<i', 0)     # TODO ?
-        binBars         = pack('<i', len(uniBars))
-        binFromDate     = pack('<i', int(uniBars[0][0].timestamp()))     # TODO Date of first bar
-        binToDate       = pack('<i', int(uniBars[1][0].timestamp()))     # TODO Date of last bar
-        binModelQuality = pack('<d', 0)     # TODO ?
+        # Build header (728 Bytes in total)
+        header = bytearray()
+        header += pack('<i', 405)                                                       # Version
+        header += bytearray('Copyright 2001-2015, MetaQuotes Software Corp.'.ljust(64,  # Copyright
+                            '\x00'), 'latin1', 'ignore')
+        header += bytearray('FxPro.com-Demo04FixedSpread'.ljust(128,                    # Server
```

Comment:
This option should be configurable and it's basically the profile name of the folder in `history/` which expect to exist, based on `Server` value from ini file. Maybe set as `-s` argument (as Server), and rename existing `-s` for Symbol, to `-c` for Currency Pair. Set default value to: `default`.